pokemongo_bot/cell_workers/recycle_items.py

```diff
@@ -3,10 +3,12 @@
 from pokemongo_bot.base_task import BaseTask
 from pokemongo_bot.tree_config_builder import ConfigException
 
+
 class RecycleItems(BaseTask):
     SUPPORTED_TASK_API_VERSION = 1
 
     def initialize(self):
+        self.min_inventory_items_to_run = self.config.get('min_inventory_items_to_run', 0)
```

Comment:
Instead of the minimum number of items we have to have, maybe it should be the number of slots we have left in the bag. That way if people purchase bag upgrades, they don't need to change their config.